engine/apps/labels/tests/factories.py

```diff
@@ -10,15 +10,15 @@
 
 
 class LabelKeyFactory(factory.DjangoModelFactory):
-    id = UniqueFaker("sentence", nb_words=3)
+    id = UniqueFaker("pystr", max_chars=36)
```

Comment:
There is a max length limit for IDs in the cache model; force that in tests generated values ([failing test](https://github.com/grafana/oncall/actions/runs/6827306305/job/18569140244?pr=3322#step:5:644))